packages/server/docker-compose.yml

```diff
@@ -1,14 +1,27 @@
 version: '3.4'
 
 volumes:
-  db-data:
+  pioneer-db-data:
+
+networks:
+  default:
+    name: joystream_default
+    external: true
 
 services:
-  db:
+  pioneer-api:
+    image: thesan/pioneer-backend # TODO change this to joystream/...
+    build:
+      context: ../..
+      dockerfile: backend.Dockerfile
+    ports: ['${PORT}:${PORT}']
+    depends_on: [pioneer-db]
+    env_file: [.env] # ensure `.env` exist
+    environment:
+      DATABASE_URL: postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@pioneer-db:5432/${POSTGRES_DB}
+
+  pioneer-db:
     image: postgres:12
```

Comment:
Haven't noticed earlier but I think we could use a newer version? 12 is 4 years old already, 16 was released recently, wondering if it would work?

Reply:
Right it should I'll set it to 16